localgovdrupal/localgov_workflows
# allow installing of localgov_workflows from config import

Recreate pull request from https://github.com/Polynya https://github.com/localgovdrupal/localgov_workflows/pull/64 to allow installing of localgov_workflows from config import, where assumed dependencies might not be available at install time.



## What does this change?

Changes the way the localgov_workflows atemtps to add the localgov_editorial workflow to all new content  types.

## How to test

Good question! 

Here's one way to generate an error in a standard install, without this branch.

1. Install a new localgov_drupal site
2. Export database backup
3. Enable localgov_workflows and localgov_guides
4. Export config
5. Import db 
6. Import config

Or more specifically: 

1. install localgov profile

```
ddev drush sql-dump > db.sql
ddev drush en localgov_workflows localgov_guides
ddev drush cex -y
ddev drush sql-drop -y
ddev drush sqlc < db.sql -y
ddev drush cim -y
```

See this error: 

```
 [notice] Synchronized extensions: install workflows.
 [notice] Synchronized extensions: install content_moderation.
 [notice] Synchronized extensions: install diff.
 [notice] Synchronized extensions: install localgov_guides.
 [notice] Synchronized extensions: install responsive_preview.
 [notice] Synchronized extensions: install scheduled_transitions.

In EntityType.php line 916:
                                                                                     
  Missing bundle entity, entity type node_type, entity id localgov_guides_overview.  
                                                                                     
```


## How can we measure success?



## Have we considered potential risks?

After this change, some content types might not get the localog_editorial workflow added automagically.
